pkg/internal/codegen/parse/crd.go

```diff
@@ -577,6 +608,31 @@ func getValidation(comment string, props *v1beta1.JSONSchemaProps) {
 		}
 	case "Format":
 		props.Format = parts[1]
+	case "AnyOf[]":
+		anyof := []v1beta1.JSONSchemaProps{}
+		intialValues := strings.Split(parts[1], ";")
+		for _, s := range intialValues {
+			value := strings.Split(s, ",")
+
+			if len(value) < 2 {
+				log.Fatalf("please enter the annotation in type:<value>,format:<value> %s:", comment)
+			}
+			typeValue := value[0]
```

Comment:
shouldn't we actually be parsing the sub-value out of here instead of assuming that type is first, etc?

Reply:
Ack. I've handled this logic in the code